File: hotel_search/v2/by_hotel/Python SDK/hotel_search.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    # Get list of offers for a specific hotel
    hotel_offers = amadeus.shopping.hotel_offers_by_hotel.get(hotelId = 'HSMADAMI')
    # print(hotel_offers.data)
except ResponseError as error:
    raise error

File: hotel_search/v2/by_offer/iOS SDK/HotelSearch.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

let hotelId = "176383FB301E78D430F81A6CB6134EBF801DCC1AE14FC9DCCE84D17C6B519F5B"

amadeus.shopping.hotelOffer(hotelId: hotelId).get(
    onCompletion: {
        response, error in
        if error == nil {
            print(response!.data)
        }
                                        })
